libs/langchain/langchain/llms/databricks.py

```diff
@@ -137,6 +137,7 @@ def get_default_api_token() -> str:
 
 
 class Databricks(LLM):
```

Comment:
@dbczumar We already have `Databricks`.

Reply:
@harupy Have we tested it with Foundation Model endpoints, external model endpoints to make sure that it works?

Reply:
Nope, should we keep this and create a new one with the Databricks deployments client?

Reply:
We should make this class compatible with Foundation Model endpoints, external model endpoints. We shouldn't duplicate it.